infrasec/tutorials/atlantis.md

```diff
@@ -0,0 +1,448 @@
+
+1. Create the new certificate in GovCloud using the [`aws_acm_certificate` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate)
+2. Export the values as an output in the `output.tf` file:
+
+    ```hcl
+    output "atlantis_cert_validation" {
+      description = "Map of data for atlantis. cert validation"
+      value       = aws_acm_certificate.acm_atlantis_external.domain_validation_options
+    }
+    ```
+
+    Once we've merged the PR, we'll have the cert validation output value to validate the certificate in the next step.
+
+3. Validate the Route53 DNS records in the Commercial account using the [`aws_acm_certificate_validation` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate_validation).
+
+    We'll need to find the `arn` to plug into the [atlantis module](https://registry.terraform.io/modules/terraform-aws-modules/atlantis/aws/latest) we'll call in the next step. After merging the PR to add the certificate, we can find this `arn` in the console's Certificate Manager under the "Details" section for the domain name (in our example, `atlantis.exp.net`).
+
+### Set up the Image for Fargate to Use 🐋
+
+For any project not requiring a specific type of docker implementation, we can simply [pass in the `trussworks-atlantis-ecs-image` image](https://github.com/trussworks/trussworks-atlantis-ecs-image) when we make the module call in [Step 3](#step-3-call-the-atlantis-module-).
+
+If we do not set the `atlantis_image` variable, we'll find `atlantis:latest` is used by default. This default is not recognized as updated when a new "latest" is released due to the word remaining unchanged. Therefore, we recommend passing in a numbered version of the Atlantis docker image to the `atlantis_image` var.
+
+## Step 3: Call the [Atlantis module](https://registry.terraform.io/modules/terraform-aws-modules/atlantis/aws/latest) 🧜
+
+We'll start with the most basic Atlantis module call we can. Assuming an existing VPC, Robot user, validated certificate, zone name, Atlantis Docker image, and GitHub connection, the code will look something like this
+
+  ```hcl
+  module "atlantis" {
+    source  = "terraform-aws-modules/atlantis/aws"
+    version = "~> 2.36.0" // make sure to update to the latest version
+
+    name = "${var.environment}-atlantis"
+
+    atlantis_image = "milmove/circleci-docker:milmove-atlantis"
```

Comment:
This probably isn't what we want in the example (use the truss image or the generic atlantis image).